pkg/types/provider_llo.go

```diff
@@ -1,11 +1,21 @@
 package types
 
 import (
+	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+
 	"github.com/smartcontractkit/chainlink-common/pkg/types/llo"
 )
 
+type LLOConfigProvider interface {
+	OffchainConfigDigester() ocrtypes.OffchainConfigDigester
+	// One instance will be run per config tracker
+	ContractConfigTrackers() []ocrtypes.ContractConfigTracker
+}
```

Comment:
Don't we need GRPC/protos for this?

Reply:
Oh, I didn't realize these are only superficially connected to the Relayer interfaces :thinking:

Reply:
Do we? I don't know how that would work. We always returned a contract config tracker here

Reply:
If we had support for the rest, then this wouldn't compile until you filled in the new ones, but we just have it stubbed out to error I guess :shrug: